### PR TITLE
Libpitt/218 group organs

### PR DESCRIPTION
**Pull request by libpitt**
Change log:
1. Add methods `getHierarchy`,` makeHierarchyFilters`  for determining parent category and respective filters 
2. Use `filteredValue` prop instead to keep filter selected
3. Renamed `defaultFilteredValue` and `defaultSortOrder` to `urlParamFilters` and `urlSortOrder` respectively avoid confusion and to indicate source of filtering and order data
4. Update example envs so that field index names found therein will be handled during definition of `urlParamFilters`
5. Remove deprecated react `defaultProps`

**Please update to:**
SN: `NEXT_PUBLIC_DEFAULT_FILTER_FIELDS = '["group_name", "status", "organ", "dataset_type", "source_type", "has_rui_info", "assigned_to_group_name"]'`
HM: `NEXT_PUBLIC_DEFAULT_FILTER_FIELDS = '["group_name", "status", "organ", "dataset_type", "has_rui_info", "assigned_to_group_name"]'`

### CODE REVIEW

File: src/package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "data-ingest-board",
-  "version": "1.3.10",
+  "version": "1.3.11",
   "private": true,
   "scripts": {
     "all": "npm-run-all --parallel dev css addons",
-    "dev": "next dev",
+    "dev": "next dev --port 3001",
```
Comment (maxsibilla) [inline src/package.json:7]: @libpitt I think this was accidentally pushed

**Comment by maxsibilla** [comment]
@libpitt If you filter by "Lung" or "Blood" by itself you get results but when you filter by multiple organs where there is a hierarchy it breaks the table:
image

**Comment by libpitt** [comment]
Thanks much @maxsibilla. It should be fine now.

Screen Shot 2024-08-21 at 3 47 41 PM

File: src/package.json (hunk 1 above)
Comment (libpitt) [reply src/package.json:7]: @maxsibilla  I didn't accidentally push it. But can revert it if you think that is better. Portal runs on default `3000`, and to have them running at the same time better this on another port; I always have to change it whenever I work on this project. Just committed it indefinitely.

Comment (maxsibilla) [reply src/package.json:7]: @libpitt Thats fair, it doesn't interfere with the docker production build and deployment so I am ok with this then

**Review by maxsibilla — APPROVED** [review]
@yuanzhou This PR addresses the change that Jonathan requested without having to modify the data status responses in the SenNet and HuBMAP Ingest APIs

**Review by yuanzhou — APPROVED** [review]

**Pull request merged**